resotolib/test/test_utils.py

```diff
@@ -372,3 +373,40 @@ def merge_fn(existing, update):
         return update
 
     assert merge_json_elements(default, update, merge_fn) == {"b": {"ba": 3}, "c": 2}
+
+
+def test_drop_deleted_attributes():
```

Comment:
Can we create a test that includes:
- defined config in json with overrides
- a config (attrs) class that has one new entry not in the json config as well as one deleted
--> config should be updated

- defined config in json with overrides
- a config (attrs) class with the same properties as the config
--> no update

Reply:
done